beets/util/__init__.py

```diff
@@ -776,16 +776,20 @@ def interactive_open(targets, command):
 
     Can raise `OSError`.
     """
-    # Split the command string into its arguments.
-    try:
-        command = shlex_split(command)
-    except ValueError:  # Malformed shell tokens.
-        command = [command]
-    command.insert(0, command[0])  # for argv[0]
+    if command:
```

Comment:
This construction looks like it could be simplified by simply redefining the function as

``def interactive_open(targets, command = interactive_open()):``